pkg/specgen/generate/container.go

```diff
@@ -136,8 +136,14 @@ func CompleteSpec(ctx context.Context, r *libpod.Runtime, s *specgen.SpecGenerat
 			return nil, fmt.Errorf("unable to process variables for --env-merge %s: %w", e, err)
 		}
 		splitWord := strings.Split(processedWord, "=")
```

Comment:
I'm sure I'm missing something but from what I've tested, the imagebuilder's shell parser, for its purposes, doesn't allow for a value within `${..}` so this would always error out but I agree, it should be a precise split, I've tried using Cut() here, which seems to suit the use-case, ptal.

Reply:
This doesn't work right now:
```
$ podman run --env-merge 'PATH=$PATH:path/with=/equal' --rm alpine printenv PATH
/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:path/with
```
using Cut() should work fine here